app/generator/get-document-definition.js

```diff
@@ -1,17 +1,24 @@
-const styles = require('./styles')
-const { generateContent } = require('./content')
 const { A4 } = require('./page-sizes')
+const style = require('./print-styles')
+const { generateContent } = require('./content')
 const { millimetresToPoints } = require('./conversion')
+const leftMargin = 15
+const topMargin = 5
+const rightMargin = 15
+const bottomMargin = 5
```

Comment:
pedantic but could we just combine these into verticalMargin and horizontalMargin? Saves us two constant definitions?